src/services/transactions/updateSignedTransactions.ts

```diff
@@ -33,3 +35,10 @@ export function setTransactionsDisplayInfoState(
 ) {
   store.dispatch(setTransactionsDisplayInfo(payload));
 }
+
+export function updateSignedTransactionCustomTransactionInformationState(payload: {
```

Comment:
I think it would be better if this was a hook

Reply:
Why ? This is just an utils function that updates a specific field in the store